routes/cube/deck.js

```diff
@@ -284,6 +284,71 @@ router.get('/download/cockatrice/:id/:seat', async (req, res) => {
   }
 });
 
+router.get('/download/topdecked/:id/:seat', async (req, res) => {
+  try {
+    const deck = await Draft.getById(req.params.id);
+    if (!deck) {
+      req.flash('danger', `Deck ID ${req.params.id} not found/`);
+      return res.redirect('/404');
+    }
+    const seat = deck.seats[0];
+
+    const DECK_CSV_HEADER = 'Quantity,"Name",Scryfall,Zone';
+
+    res.setHeader('Content-disposition', `attachment; filename=${seat.title.replace(/\W/g, '')}.csv`);
+    res.setHeader('Content-type', 'text/plain');
+    res.charset = 'UTF-8';
+
+    res.write(`${DECK_CSV_HEADER}\r\n`);
+
+    const main = {};
+    const mainCardID = {};
+    for (const row of seat.mainboard) {
+      for (const col of row) {
+        for (const cardIndex of col) {
+          const card = deck.cards[cardIndex];
+          const details = carddb.cardFromId(card.cardID);
+          const name = `${details.name}`;
```

Comment:
I recommend using oracle_id over name, generally better practice

Reply:
(there are some cards that share a name, but oracle id is unique for each game piece)